databases/relational-databases.md

```diff
@@ -31,6 +32,35 @@ To understand, query, and insert information into a relational database, a techn
 
 ### Specific Things To Teach
 
+- The specific problems a database solves
+- Users of databases
+- Structure of a relational database; tables with columns and references to other tables
+- Few basic column types
+- The four basic SQL verbs and their use
+- Bonus topic for the advanced: foreign key constraints
+- Explain different databases with slightly different dialects (MySQL - PostgreSQL)
+- Explain existence of other types of non-relational, non-sql databases that will be encountered
+
+### Materials
+
+#### Education
+
+- [Khan Academy's introductory SQL, videos](https://www.khanacademy.org/computing/computer-programming/sql/sql-basics/v/welcome-to-sql)
+- [TutorialsPoint, detailed](https://www.tutorialspoint.com/sql/)
```

Comment:
Where is this in the outline? If its not there, can you add it under ### Lesson?  Same with the link in line 50.